docs/DeveloperGuide.adoc

```diff
@@ -275,55 +275,319 @@ Refer to the guide <<Testing#, here>>.
 Refer to the guide <<DevOps#, here>>.
 
 [appendix]
+//tag::productScope[]
 == Product Scope
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* has a need to record expenses and income
+* has a need to keep to a certain budget every month
+* has a need to record debts
+* want to be reminded of his/her own debts
+* wants to remind his/her friends to pay back their debts
+* has a lot of friends to keep track in address/debt book
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* is reasonably comfortable using <<CLI, CLI>> apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: records expenses/debts faster than a typical mouse/GUI driven app
+//end::productScope[]
 
 [appendix]
+//tag::userStories[]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a/an ... |I want to ... |So that ...
+|`* * *` |university student that buys a lot of stuff |keep track of my spending |I do not overspend
 
-|`* * *` |user |add a new person |
+|`* * *` |student with fixed monthly allowance |track my spendings in a month |I will make sure I save money every month
```

Comment:
User profile does not really match desired functionality and expected outcome. 

Everyone with basic financial literacy would want to _track_ their spendings. It makes more sense for someone with a limited budget to, well, _limit_ their spendings.

Furthermore, tracking spending and aiming to save money don't directly link up either.